src/library-authoring/author-library/LibraryAuthoringPage.jsx

```diff
@@ -434,6 +485,69 @@ const LibraryAuthoringPageHeader = connect(
   },
 )(injectIntl(LibraryAuthoringPageHeaderBase));
 
+const ContentTagsDrawer = ({ openContentTagsDrawer, setOpenContentTagsDrawer }) => {
+  if (openContentTagsDrawer) {
+    document.body.classList.add('drawer-open');
+  } else {
+    document.body.classList.remove('drawer-open');
+  }
+
+  useEffect(() => {
+    const handleCloseMessage = (event) => {
+      if (event.data === 'closeManageTagsDrawer') {
+        setOpenContentTagsDrawer('');
+      }
+    };
+
+    const handleCloseEsc = (event) => {
+      if (event.key === 'Escape' || event.keyCode === 27) {
+        setOpenContentTagsDrawer('');
+      }
+    };
+
+    // Add event listener to close drawer when close button is clicked or ESC pressed
+    // from within the Iframe
+    window.addEventListener('message', handleCloseMessage);
+    // Add event listern to close the drawer when ESC pressed and focus outside iframe
+    // If ESC is pressed while the Iframe is in focus, it will send the close message
+    // to the parent window and it will be handled with the above event listener
+    window.addEventListener('keyup', handleCloseEsc);
+
+    return () => {
+      window.removeEventListener('message', handleCloseMessage);
+      window.removeEventListener('keyup', handleCloseEsc);
+    };
+  }, []);
+
+  // TODO: The use of an iframe in the implementation will likely change
+  const renderIFrame = () => (
+    openContentTagsDrawer
+      ? (
+        <iframe
+          title="manage-tags-drawer"
+          src={`${getConfig().COURSE_AUTHORING_MFE_BASE_URL}/tagging/components/widget/${openContentTagsDrawer}`}
+          frameBorder="0"
+          style={{ width: '100%', height: '100%' }}
+        />
+      )
+      : null
+  );
+
+  return (
```

Comment:
```suggestion
  return openContentTagsDrawer && (
```
If you render conditionally here, you can remove the code that hides the div using style